ui/src/app/core/gcs.service.ts

```diff
@@ -7,30 +7,21 @@ declare const gapi: any;
 @Injectable()
 export class GcsService {
 
-  private static readonly apiPrefix = "https://www.googleapis.com/storage/v1"
+  private static readonly apiPrefix = "https://www.googleapis.com/storage/v1";
+  private static readonly maximumBytes = "1000000";
```

Comment:
opt: This feels a bit wrong as a string.. since you're using `==` below it would still actually work if this were a number.

Alternatively, switch to number then `contentLength === String(maximumBytes)` below